google/cloud/pubsublite/CMakeLists.txt

```diff
@@ -62,11 +62,11 @@ target_link_libraries(
            google-cloud-cpp::api_annotations_protos
            google-cloud-cpp::api_http_protos)
 
-file(
-    GLOB source_files
+file(GLOB source_files
```

Comment:
while globs were fine for a purely generated library, now that we are hand-crafting things we may want to reconsider.  We can move to explicitly listing files in this PR, or open an issue to clean it up later, your call, I am fine either way.

Reply:
Ack, I'll clean up in a follow up